contents/codes/511.md

```diff
@@ -26,14 +26,14 @@ Unknown clients then have all traffic blocked, except for that on TCP port 80, w
 
 For example, a user agent might connect to a network and make the following HTTP request on TCP port 80:
 
-```
+```plain
 GET /index.htm HTTP/1.1
 Host: www.example.com
 ```
 
 Upon receiving such a request, the login server would generate a 511 response:
 
-```
+```html
```

Comment:
btw thank you for these changes as well, nice catch!

Reply:
Added a commit in which I removed the ```unlisted``` option.